Sources/RSA.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  RSA.swift
+//  OAuthSwift
+//
+//  Created by Noam Bar-on on 6/19/17.
+//  Copyright © 2017 Dongri Jin. All rights reserved.
+//
+
+import Foundation
+import SwiftyRSA
```

Comment:
So OAuthSwift become dependant on SwiftyRSA
So OAuthSwift.podspec and Package.swift must be updated

But for carthage a Cartfile must be created, and see the comment on Podfile